### ISSUE TITLE
subtitle issue

### ISSUE THREAD

**Issue by sajinman** [mpv-player/mpv#13101]
### Important Information
- mpv version:
v0.37.0-94-g3250f6e4
- Platform and Version:
Windows 11 23H2
- Source of the mpv binary:
https://sourceforge.net/projects/mpv-player-windows/files/release/

Subtitle file lines:
```
14:26.000 --> 14:33.390
According to my father, she said "trouble's 
here" after I was born and then was gone.

14:33.390 --> 14:39.900
<b>Mother: 


Father:</b>

14:33.390 --> 14:39.900
<b>Trouble (Kuro) is here.
Kuro (9th son) is here.</b>

14:33.700 --> 14:36.000
But my father misheard her,
```
### Expected behavior
subtitle should continue

### Actual behavior
subtitle stops

### Log file
[output.txt](https://github.com/mpv-player/mpv/files/13683729/output.txt)

**Comment by llyyr** [comment]
I'm not sure where you got your file from but that _feels_ like an ASS file terribly converted to VTT. This isn't a bug with anything, the file is just broken, you're not allowed to have empty lines within a VTT cue. See: https://www.w3.org/TR/webvtt1/#introduction-multiple-lines

ffmpeg thinks the file ends at this block because of the empty lines.
```
14:33.390 --> 14:39.900
<b>Mother: 


Father:</b>
```

**Comment by sajinman** [comment]
can something be done about it so that mpv continues showing subtitle?

**Comment by llyyr** [comment]
> can something be done about it so that mpv continues showing subtitles in these situations?

No, fix whatever software generated these broken subtitles. Potplayer's behavior is broken

**Comment by sajinman** [comment]
i didn't convert it. i downloaded this vtt from aniwatch.to. please help me . i need mpv
